src/cljs/rems/application.cljs

```diff
@@ -339,8 +370,8 @@
                 :accept ".pdf, .doc, .docx, .ppt, .pptx, .txt, image/*"
                 :class (when validation "is-invalid")
                 :disabled readonly
-                :on-change (set-attachment id)}]
-       [:button.btn.btn-secondary {:on-click (fn [e] (.click (.getElementById js/document (id-to-name id))))} (text :t.form/upload)]])
+                :on-change (set-attachment id title)}]
+       [:button.btn.btn-default {:on-click (fn [e] (.click (.getElementById js/document (id-to-name id))))} (text :t.form/upload)]])
```

Comment:
There is no `.btn-default` in Bootstrap 4. It has been renamed to `.btn-secondary`. https://getbootstrap.com/docs/4.0/migration/#buttons

Reply:
The available button classes are listed in https://getbootstrap.com/docs/4.1/components/buttons/